src/scripts/main.js

```diff
@@ -5,4 +5,43 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const list = document.querySelector('.dashboard');
+
+for (let i = 0; i < people.length; i++) {
+  const line = document.createElement('tr');
+  const nameLine = document.createElement('td');
+
+  nameLine.textContent = people[i].name;
+  line.append(nameLine);
+
+  const genderLine = document.createElement('td');
```

Comment:
```suggestion
  const genderCell = document.createElement('td');
```

fix it everywhere, because "td" element defines a cell of a table